stable/sentry/templates/service.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+{{- if .Values.service.enableInternalLB }}
+  annotations:
+    service.beta.kubernetes.io/azure-load-balancer-internal: "true"
```

Comment:
I think the annotations should simply be a templated value, with this example commented in `values.yaml`. See https://github.com/kubernetes/charts/blob/7bad0e00872159e4ff3c3774f44ac9fe3b08e222/incubator/gogs/values.yaml#L228 for example.

Reply:
fair enough.

Reply:
Please fix his so we can merge the PR.